tools/build/cross-build/include/mac/sys/_types.h

```diff
@@ -41,5 +41,3 @@
  * __darwin_ct_rune_t exists.
  */
 typedef __darwin_ct_rune_t __ct_rune_t;
-/* Needed for opensolaris compat. */
-typedef	__int64_t	off64_t;
```

Comment:
I don't understand this one...

Reply:
Moved to common given it’s needed for musl, it seems

Reply:
Exactly